python/ray/autoscaler/v2/sdk.py

```diff
@@ -63,7 +63,7 @@ def request_cluster_resources(
 
 
 def get_cluster_status(
-    gcs_address: Optional[str] = None, timeout: int = DEFAULT_RPC_TIMEOUT_S
```

Comment:
this is the one.

Reply:
Q: can we just completely remove the code that uses runtime context to avoid ray.init inside a autoscaler?

Or alternaitvely, when you do ray.init inside an autoscaler, you should always do it with internal namespace

Reply:
Removed the code that initialzies runtime context (bascially when creating GcsClient when address missing) from sdk compelely.